projects/batfish/src/main/java/org/batfish/representation/juniper/JuniperConfiguration.java

```diff
@@ -224,6 +231,10 @@ private BgpProcess createBgpProcess(RoutingInstance routingInstance) {
         mg.setLocalAs(routingInstanceAs);
       }
     }
+    // Set default authentication algorithm if missing
+    if (mg.getAuthAlgorithm() == null) {
+      mg.setAuthAlgorithm(BgpAuthenticationAlgorithm.HMAC_SHA_1_96);
```

Comment:
Please refactor this to a constant. That is, make a constant called `DEFAULT_BGP_KEY_CHAIN_AUTHENTICATION_ALGORITHM` set to `BgpAuthenticationAlgorithm.HMAC_SHA_1_96`, and use that constant here.